src/citrine/seeding/dataset_deletes.py

```diff
@@ -0,0 +1,53 @@
+"""Functions that perform dataset deletes."""
+from typing import List, Tuple
+
+from citrine.resources.api_error import ApiError
+from citrine.resources.dataset import Dataset
+
+from gemd.entity.link_by_uid import LinkByUID
+
+
+def wipe_dataset(dataset: Dataset, delete_templates: bool = False) -> \
+        List[Tuple[LinkByUID, ApiError]]:
+    """
+    Wipes the GEMD objects from a dataset.
+
+    Parameters
+    ----------
+    dataset : Dataset
+        The dataset to remove GEMD objects from
+    delete_templates : bool, optional
+        An option to also remove templates from the given dataset, by default False
+
+    Returns
+    -------
+    List[Tuple[LinkByUID, ApiError]]
+        A list of (LinkByUID, api_error) for each failure to delete an object.
+        Note that this method doesn't raise an exception if an object fails to be
+        deleted.
+
+    """
+    print("Collecting GEMDs...")
```

Comment:
This should really be done in logging, but it's consistent with other elements of the `seeding` library, so okay for now.